Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -127,8 +238,124 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
           find_instance_method_call(body_node, "ENV", :x11) do
             problem 'Use "depends_on :x11" instead of "ENV.x11"'
           end
+
+          find_every_method_call_by_name(body_node, :assert).each do |method|
+            if method_called?(method, :include?) && !method_called?(method, :!)
+              problem "Use `assert_match` instead of `assert ...include?`"
+            end
+          end
+
+          find_every_method_call_by_name(body_node, :depends_on).each do |method|
+            next unless method_called?(method, :new)
+            problem "`depends_on` can take requirement classes instead of instances"
+          end
+
+          os = [:leopard?, :snow_leopard?, :lion?, :mountain_lion?]
+          os.each do |version|
+            find_instance_method_call(body_node, "MacOS", version) do |method|
+              problem "\"#{method.source}\" is deprecated, use a comparison to MacOS.version instead"
+            end
+          end
+
+          find_instance_method_call(body_node, "Dir", :[]) do |method|
+            path = parameters(method).first
+            next if !path.str_type?
+            next unless match = regex_match_group(path, /^[^\*{},]+$/)
+            problem "Dir([\"#{string_content(path)}\"]) is unnecessary; just use \"#{match[0]}\""
+          end
+
+
+          fileUtils_methods= Regexp.new(FileUtils.singleton_methods(false).map { |m| '(?-mix:^'+Regexp.escape(m)+'$)' }.join '|')
```

Comment:
I know it's fussy, but for consistency there should probably be a space between the `_methods` and the `=` here?

Reply:
Its fine. Thanks! 👍

Reply:
And don‘t use camelCase in Ruby.